### PR TITLE
Remove sub-type from the form

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?
It has been removed from the design of this page and has been moved to the starting dialog

### CODE REVIEW

**Review by mjgiarlo — COMMENTED** [review]
Tests failing and a question: did you run:

```
bundle exec rake rails_rbi:all
bundle exec srb rbi hidden-definitions
bundle exec srb rbi suggest-typed
```

:question:

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# typed: strict
```
Comment (jcoyne) [inline app/models/user.rb:1]: @mjgiarlo I think `bundle exec srb rbi suggest-typed` did this.  Do we want this?

Comment (mjgiarlo) [reply app/models/user.rb:1]: @jcoyne It probably thinks it's helping us. To test that, you can try flipping that back to `typed: strict` and running `srb tc` to see if anything breaks?

Comment (mjgiarlo) [reply app/models/user.rb:1]: Also, it seems weird that sorbet-rails removed so many definitions. but whatever?

**Review by mjgiarlo — APPROVED** [review]

**Pull request merged**